### PR TITLE
Mod Tools

### PR DESCRIPTION
**Pull request by funkydemir66**
Mod Tools for G-Earth

### CODE REVIEW

**Comment by funkydemir66** [comment]
dude... i finally opened it

File: store/extensions/Mod Tools/extension.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Python",
+      "version": "2.0"
+    },
+
+    "language": "Python",
+    "commands": :mt pla/:mt mute (minute)/:mt rmute {
```
Comment (Alynva) [inline store/extensions/Mod Tools/extension.json]: something is wrong here

**Review by Alynva — CHANGES_REQUESTED** [review]
the extension should be in a `.zip` file, instead of `.rar`

**Comment by Alynva** [comment]
Also, that's not an screenshot

File: store/extensions/Mod Tools/extension.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "default": ["python", "ModTools.py", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"]
+    },
+    
+    "compatibility": {
+        "systems": ["Linux", "Windows", "Mac"],
+        "clients": ["Flash"]
+    },
+
+    "submissionDate": "03-04-2002 13:38:00",
+    "updateDate": "03-04-2002 13:38:00",
```
Comment (WiredSpast) [inline store/extensions/Mod Tools/extension.json]: What year is it? xD

File: store/extensions/Mod Tools/extension.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "version": "2.0",
+    "categories": ["Moderation", "Others", "Fun"],
+    
+    "source": "https://github.com/funkydemir66/ModTools",
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Python",
+      "version": "2.0"
```
Comment (WiredSpast) [inline store/extensions/Mod Tools/extension.json]: Also this version of G-Python doesn't appear to exist

File: store/extensions/Mod Tools/extension.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+    "title": "Mod Tools",
+    "description": "can be used to mute or remove commands: :mt pla/:mt mute (minute)/:mt rmute ",
+    "authors": [
+      {
+        "name": "26",
+        "hotel": ".com.tr",
+        "username": "funkydemir66",
+        "discord": "26#7305",
+        "twitter": ""
```
Comment (WiredSpast) [inline store/extensions/Mod Tools/extension.json]: Also it's better to just leave out the twitter line instead of given it an empty string

**Comment by WiredSpast** [comment]
I've corrected the extension.json file for you so you have an example for next time, but you will have to replace the `extensions.rar` file with a `extension.zip` file yourself

**Comment by funkydemir66** [comment]
I'm freaking out :D

**Comment by WiredSpast** [comment]
image
image

**Comment by WiredSpast** [comment]
extension.zip not extensions.zip and the .rar file has to be removed from github

**Comment by funkydemir66** [comment]
I'll take care of it and deliver it, I'm at the library right now

**Comment by funkydemir66** [comment]
i make extension.zip

**Comment by funkydemir66** [comment]
I think everything is ok except the screenshot.

thanks for your help i will do it as soon as possible :P

**Comment by WiredSpast** [comment]
- You might want to fix a small big as well, when you enter something that isn't a number with the `:mt mute ...` command the extension will crash and the user's entire client will get stuck.
image


- Also I would recommend to use either one of these 2 packages (they get send when you click on a user) to identify the user instead of the Unmute user button within the ambassador tools, because if someone has Client-side staff permissions disabled they won't have access to the ambassador tools
image

**Comment by funkydemir66** [comment]
A few hours later v/

**Comment by funkydemir66** [comment]
ayrıca (tıkladığınızda gönderilirler) bir kullanım etkili değildir

anyway thanks for the advice

**Comment by funkydemir66** [comment]
okey is finished

**Comment by funkydemir66** [comment]
there are a few mistakes for now i will tell you when i get it done

**Comment by funkydemir66** [comment]
ok work is done all bugs is fixed

**Comment by WiredSpast** [comment]
`extension.zip` not a loose py file 
image

**Comment by WiredSpast** [comment]
Also maybe add a explanation of what the commands do exactly either in the description part of the `extension.json` file or a readme in the repo, because it's really not clear what they would do

**Comment by funkydemir66** [comment]
okey i upgraded

File: store/extensions/Mod Tools/extension.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+    "title": "Mod Tools",
+    "description": "can be used to mute or remove commands: :mt pla/:mt mute (minute)/:mt rmute. User guide: write :mt pla and click player, write :mt mute (minute) or :mt rmute
+    ",
```
Comment (Alynva) [inline store/extensions/Mod Tools/extension.json]: remove this line break

**Comment by WiredSpast** [comment]
If you want a linebreak in the actual page you have to use `\n`

**Comment by funkydemir66** [comment]
okey

**Comment by funkydemir66** [comment]
its finished

**Pull request merged**